merlin/systems/dag/ops/workflow.py

```diff
@@ -156,47 +99,4 @@ def export(
         backend: str = "ensemble",
     ):
         """Create a directory inside supplied path based on our export name"""
-        modified_workflow = self.workflow.remove_inputs(self.label_columns)
-        export_name = self.__class__.__name__.lower()
-        node_name = f"{node_id}_{export_name}" if node_id is not None else export_name
-
-        node_export_path = pathlib.Path(path) / node_name
-        node_export_path.mkdir(parents=True, exist_ok=True)
-
-        backend_model_config = generate_nvtabular_model(
-            modified_workflow,
-            node_name,
-            node_export_path,
-            sparse_max=self.sparse_max,
-            max_batch_size=self.max_batch_size,
-            cats=self.cats,
-            conts=self.conts,
-        )
-
-        if self._python:
-            self_params = {
-                "nvt_model_name": f"{node_id}_{export_name}",
-            }
-            python_config = super().export(
-                path, input_schema, output_schema, self_params, node_id, version
-            )
-
-            return python_config
-        else:
-            return backend_model_config
-
-    @property
-    def export_name(self):
-        """
-        Provides a clear common english identifier for this operator.
-
-        Returns
-        -------
-        String
-            Name of the current class as spelled in module.
-        """
-        cls_name = self.__class__.__name__.lower()
-        if self._python:
-            return f"py_{cls_name}"
-        else:
-            return cls_name
+        raise NotImplementedError
```

Comment:
I assume this is required because the base class would go ahead an export if we didn't raise an error here, right? 👍🏻